Sprites feos de armas adios, nuevos sprites feos de armas saludos.

SagaStars opened the pull request:
- Se cambian las sprites de la energy gun y laser gun.
- Cambio a la sprite de el riot helmet y cambio de color de la pantalla azul del bulletproof helmet a rojo.
## Why It's Good For The Game
No creo que tenga que explicar por que cambiar las sprites sea algo positivo. :vomit:

![PR](https://user-images.githubusercontent.com/55407528/98999227-d1d2fc80-250d-11eb-8dd1-c553d631b53e.png)

## Changelog
:cl:
imageadd: Cambios a las sprites de armas de: lasergun y energygun.
imageadd: Nuevo riot helmet y cambio de color al bulletproof helmet.
/:cl:

SagaStars committed:
SADDEST PEOPLE FART THE LOUDER

Cokecolagua reviewed (commented):
ronaldinho soccer

Evankhell561 reviewed (changes requested):
puedes por favorcito hacer los sprites a parte para poder juzgarlos sin tener que considerar tus cambios al balance? se te agradece la gerencia.

SagaStars committed:
Update beams.dm

SagaStars committed:
Update energy.dm

SagaStars committed:
Update energy.dm

SagaStars committed:
Update laser.dm

SagaStars commented:
> puedes por favorcito hacer los sprites a parte para poder juzgarlos sin tener que considerar tus cambios al balance? se te agradece la gerencia.

Aqui va solo con los cambios de sprites.

Evankhell561 commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -74,9 +74,10 @@
+	item_state = "swat"
 	armor = list("melee" = 15, "bullet" = 60, "laser" = 10, "energy" = 10, "bomb" = 40, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
 	dog_fashion = null
+	hispania_icon = TRUE
```
Esto no es suficiente para que se visualicen correctamente. Sigue el ejemplo de spricen en su pr de arreglar la visualización de la ropa en los vending.

SagaStars replied:
Arreglao

Evankhell561 commented on code/modules/projectiles/guns/energy/nuclear.dm:
```diff
@@ -89,6 +89,5 @@
 	var/fail_tick = 0
 	charge_delay = 5
 	can_charge = 0
-	ammo_x_offset = 1
```
Why?

SagaStars replied:
Para que este acorde la municion a la sprite. Con el offset a 1 toda la municion se acumula en 4 pixeles.

SagaStars committed:
Evan fix

Evankhell561 commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -73,10 +73,12 @@
+	icon = 'icons/hispania/mob/suit.dmi'
 	icon_state = "swat"
-	item_state = "swat-alt"
+	item_state = "swat"
```
Sería mejor que tú sprite tenga el mismo nombre que el sprite de Paradise.

SagaStars replied:
Piénsalo mas como un fix, el hecho de que tuviera `-alt` solo hacia mas confuso encontrar el hat, asi es como deberia estar.

Evankhell561 replied:
Está bien pero si este cambio se revierte por x motivo será aún más difícil de reparar

SagaStars replied:
Muchos cambios de hispania ya son así, cual es la diferencia? En todo caso yo los reparare.

Evankhell561 replied:
El único que recuerdo hizo algo parecido fue silver con el cangrejo y el mismo dijo que mantener los nombres del sprite sería lo mejor. También los otros cambios de nombres de sprites han Sido fixes directamente en dmis de Paradise. Quizá me haya olvidado de uno pero en general no está bien.

SagaStars replied:
Bien tu ganas lo pondré como antes.

SagaStars committed:
Esperando el Insta-Merge

YOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

Dapple27 commented:
Sugerencias de Cambios a los Sprites

1ero.Sugiero que le añadas es dos pixeles a las partes rojas para que se vea mejor porque como ahora no me agrada del todo como se ve, no esta mal pero puede verse mejor con mi arreglito

![image](https://user-images.githubusercontent.com/43283605/99388728-680b7780-28ac-11eb-8346-3959211d6e8d.png)
(Tu version

![image](https://user-images.githubusercontent.com/43283605/99388443-f7645b00-28ab-11eb-8689-f17209990f11.png)
(Mi version

2do. Esta parte la verdad se me hacia un poco extraño y a mi parecer le falta detalle mi forma de como yo lo arreglaria sera poniendo estos detallitos simples 

![image](https://user-images.githubusercontent.com/43283605/99388669-532ee400-28ac-11eb-9655-8e3d30f3c35c.png)
(Tu version)

![image](https://user-images.githubusercontent.com/43283605/99388694-5c1fb580-28ac-11eb-9350-e28507f0cfa0.png)
(Mi version)

SagaStars committed:
Parche Marca: Danaleja

SagaStars commented:
@Danaleja2005
 Revisión de Sprites:
-------------
Se acomoda el mango como primera sugerencia presentada.
![image](https://user-images.githubusercontent.com/55407528/99394566-79a54d00-28b5-11eb-9e31-13848505943c.png)
La segunda sugerencia ha sido tomado a medias, la parte del mango en el barril se ha modificado pero no se ha hecho una separación entre el medidor de munición y el barril, se ve mucho mejor con una transición limpia sin otro relieve, no me convenció en lo absoluto. Así estaría viéndose ahora:
![image](https://user-images.githubusercontent.com/55407528/99394720-b83b0780-28b5-11eb-8dc2-22125895be2c.png)
![image](https://user-images.githubusercontent.com/55407528/99394744-c12bd900-28b5-11eb-955b-9de16320e3ba.png)

Dapple27 commented:
Se ve mejor, me sirve

![image](https://user-images.githubusercontent.com/43283605/99402408-b24a2400-28bf-11eb-8e39-7d1cf68ef34d.png)
